cms/templates/bulkupdate.html

```diff
@@ -0,0 +1,133 @@
+                  <option value="${option}" selected>${option}</option>
+                % else:
+                  <option value="${option}">${option}</option>
+                % endif
+              % endfor
+            </select>
+          </div>
+          <input type="submit" value="${_('Apply checked settings')}" class="submit-button" />
+          <input type="button" value="${_('Clear display')}" class="reset-button" disabled />
+        </form>
+      </section>
+
+      <p class="error-block"></p>
+
+      <div class="wrapper wrapper-status is-hidden">
+        <h3 class="title">${_("Bulk Update Status")}</h3>
+
+        <ol class="status-progress list-progress">
+          <li class="item-progresspoint item-progresspoint-submit is-complete">
+            <span class="deco status-visual">
+              <i class="icon fa fa-cog"></i>
+              <i class="icon fa fa-warning"></i>
+            </span>
+            <div class="status-detail">
+              <h3 class="title">${_("Applying")}</h3>
+              <p class="copy">${_("Applying your new settings")}</p>
+            </div>
+          </li>
+
+          <li class="item-progresspoint item-progresspoint-validate is-not-started">
+            <span class="deco status-visual">
+              <i class="icon fa fa-cog"></i>
+              <i class="icon fa fa-warning"></i>
+            </span>
+            <div class="status-detail">
+              <h3 class="title">${_("Validating")}</h3>
+              <p class="copy">${_("Validating your new settings")}</p>
+            </div>
+          </li>
+
+          <li class="item-progresspoint item-progresspoint-update-advanced-settings is-not-started">
+            <span class="deco status-visual">
+              <i class="icon fa fa-cog"></i>
+              <i class="icon fa fa-warning"></i>
+            </span>
+            <div class="status-detail">
+              <h3 class="title">${_("Updating Advanced Settings")}</h3>
+              <p class="copy">${_("Setting as default for future problems in Advanced Settings")}</p>
+            </div>
+          </li>
+
+          <li class="item-progresspoint item-progresspoint-update-problems is-not-started">
+            <span class="deco status-visual">
+              <i class="icon fa fa-cog"></i>
+              <i class="icon fa fa-warning"></i>
+            </span>
+            <div class="status-detail">
+              <h3 class="title">${_("Updating and Republishing Problems")}</h3>
+              <p class="copy">
+                  ${_("Applying update to all problems. Republishing published problems. This process might take longer with larger courses. You will receive an email when your update has completed.")}
+              </p>
+            </div>
+          </li>
+
+        </ol>
+      </div>
+    </article>
+
+    <aside class="content-supplementary" role="complimentary">
+      <div class="bit">
+        <h3 class="title title-3">${_("What is bulk update?")}</h3>
```

Comment:
Should this be capitalized, `Bulk Update`?

Reply:
done